src/test/java/EOorg.EOeolang/EOstrings/package-info.java

```diff
@@ -30,4 +30,4 @@
  * @checkstyle PackageDeclarationCheck (5 lines)
  * @checkstyle PackageCase (5 lines)
  */
-package EOorg.EOeolang.EOtxt;
+package EOorg.EOeolang.EOstrings;
```

Comment:
I changed the name of this package due to name conflict with `org.eolang.txt` from `eo-runtime` EO